### ISSUE TITLE
Final Project Presentation Times: Sign-Up

### ISSUE THREAD

**Issue by rickecon** [rickecon/StructEst_W19#4]
Dear MACS 40200 (Structural Estimation) students,

As a reminder, your final project write-ups are due on Friday, March 22 (midnight). These papers should be less than 20 pages (excluding appendices) and should follow the guidelines from our class discussion and notes on:
* Conclusion, introduction, and abstract [[Slides]()]
* Data description [[Slides](https://github.com/rickecon/StructEst_W19/blob/master/Projects/DataSection_slides.pdf)]
* Model description [[Slides](https://github.com/rickecon/StructEst_W19/blob/master/Projects/ModelDescr_slides.pdf)]
* Estimation/results [[Slides](https://github.com/rickecon/StructEst_W19/blob/master/Projects/EstimResults_slides.pdf)]

Each presentation will be limited to 20 minutes. Available presentation times are on Monday, March 11, and Wednesday, March 13, from 1:30 - 3:05p (see tables in [`Projects/README.md`](https://github.com/rickecon/StructEst_W19/blob/master/Projects/README.md#project-presentation-times)). Please respond to this thread to sign up for a presentation time slot.

I have also posted a LaTeX template for writing a project document in the [`Projects/LaTeXtemplates`](https://github.com/rickecon/StructEst_W19/tree/master/Projects/LaTeXtemplates) folder.

@adamoppenheimer, @avasilj, @ccerda1, @dfnoriega, @keiirizawa, @originhal, @sergioservan, @tschwieg

**Comment by sergioservan** [comment]
I would like to take the 1:55 - 2:15 on Wednesday 13

**Comment by rickecon** [comment]
OK quick change in proposals. Many of you have suggested that Monday is too soon for a presentation. Here are three alternatives, each of which involves doing seven 20-minute presentations in one continuous three-hour span.

1) Wednesday, March 13, 1:30-4:30p in Saieh 242
2) Thursday, March 14, 9:00a-12:00p, in Saieh 242
3) Friday, March 15, 9:00a-12:00p in Saieh 242

My most preferred option is #1, Wednesday, March 13. Please let me know which of those alternatives you prefer. We have to have unanimity, otherwise, I have to go back to Monday and Wednesday. If we can agree on one of the above three alternatives, then we will not hold class on Monday, March 11.

@adamoppenheimer, @avasilj, @ccerda1, @dfnoriega, @keiirizawa, @originhal, @sergioservan, @tschwieg

**Comment by originhal** [comment]
My most preferred option is 3 and follows in the order of 2>1, but all three alternatives work for me.  I wonder what our `social choice' is going to be lol.

**Comment by ccerda1** [comment]
Option 1 works for me. Best.

**Comment by avasilj** [comment]
I am holding office hours 3-4pm on Wed (which, worst case scenario, I could reschedule, though) - so my preferred options are #3 and #2.   (3>2>1)

**Comment by originhal** [comment]
So have we settled on Friday 9-12?

**Comment by rickecon** [comment]
@originhal Yes. I sent an e-mail out through Canvas. Friday morning, 9am-noon. Slots are assigned by reverse alphabetical order (see [`Project/README.md`](https://github.com/rickecon/StructEst_W19/blob/master/Projects/README.md#friday-march-15)). No class on Wednesday.
